static/js/shared/stats_var_titles.ts

```diff
@@ -210,15 +231,18 @@ const STATS_VAR_TITLES: { [key: string]: string } = {
     "Percentage of People with Mental Health Not Good",
   Percent_Person_WithPhysicalHealthNotGood:
     "Percentage of People with Physical Health Not Good",
+  "WorldBank/SH_STA_WAST_ZS": "Wasting among Children under 5",
+  "WorldBank/SH_SVR_WAST_ZS": "Severe Wasting among Children under 5",
 
   // Behaviors
   Percent_Person_SleepLessThan7Hours:
     "Percentage of People Who Sleep Less Than 7 Hours",
   Percent_Person_Obesity: "Percentage of People with Obesity",
-  Percent_Person_BingeDrinking: "Percentage of People with Binge Drinking",
+  Percent_Person_BingeDrinking: "Percentage of People That Binge Drink",
```

Comment:
along the lines of the fixes, should be lower case "that"

Reply:
Thanks for pointing out the complexity for title case... I checked all the standards in https://capitalizemytitle.com/ for "that" and they all have it capitalized... maybe we can do sentence case going forward?